apps/myaccount/src/models/validation.ts

```diff
@@ -53,4 +53,5 @@ export interface ValidationFormInterface {
     consecutiveCharacterValidatorEnabled?: boolean;
     minUniqueCharacters?: number;
     maxConsecutiveCharacters?: number;
+    enableValidator?: string;
```

Comment:
Any particular reason to define `enableValidator` type as `string`? Seems `boolean` fits.

Reply:
The API call returns a string for this. Hence leaving it as it is in the FE.